docs/source/performance/cache-backends.mdx

```diff
@@ -201,6 +201,24 @@ const server = new ApolloServer({
 });
 ```
 
+## Handling cache fetching errors
+When cache backend is unavailable, all requests depend on it will fail. It is crucial to still return successful response in this kind of scenario.
```

Comment:
I think the following paragraph covers this point nicely, so I'd recommend omitting this sentence:
```suggestion
```